packages/core/data-transfer/lib/engine/index.ts

```diff
@@ -275,15 +281,17 @@ class TransferEngine<
 
   async transferSchemas(): Promise<void> {
     const stageName: TransferStage = 'schemas';
-    const inStream = await this.sourceProvider.streamSchemas?.();
-    const outStream = await this.destinationProvider.getSchemasStream?.();
 
+    const inStream = await this.sourceProvider.streamSchemas?.();
     if (!inStream) {
-      throw new Error('Unable to transfer schemas, source stream is missing');
+      // console.log('SourceProvider did not return a schemas stream');
```

Comment:
Do we want to keep those console.log in the future?

Reply:
(same questions for the other ones below)

Reply:
I wasn't sure, we talked about leaving the logs for debugging purposes since this code should be unreachable in the import/export, but it caused output in tests which was more annoying than helpful. For now I've removed the commented logs and if want them back for some reason we can add them.